scripts/build_wasm_package.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -ex
+
+# look at DeepFilterNet/.github/workflows/build_wasm.yml for enviroment setup
+cd ./libDF/
```

Comment:
What are your politics about `cd` in scripts? Do you always run scripts from repository-root? or from `scripts` folder?